README.md

```diff
@@ -79,34 +80,27 @@ In addition to the documentation mentioned above, there is also an
 - [Ruby](https://github.com/cloudevents/sdk-ruby)
 - [Rust](https://github.com/cloudevents/sdk-rust)
 
-## Community
+## Community and Docs
 
 Learn more about the people and organizations who are creating a dynamic cloud
 native ecosystem by making our systems interoperable with CloudEvents.
 
-- Our [Governance](community/GOVERNANCE.md) documentation.
-- How to [contribute](community/CONTRIBUTING.md) via issues and pull requests.
-- [Contributors](community/contributors.md): people and organizations who helped
+- Our [Governance](docs/GOVERNANCE.md) documentation.
+- [Contributing](docs/CONTRIBUTING.md) guidance.
+- [Roadmap](docs/ROADMAP.md)
+- [Adopters](https://cloudevents.io/) - See "Integrations".
+- [Contributors](docs/contributors.md): people and organizations who helped
   us get started or are actively working on the CloudEvents specification.
-- [Demos & open source](community/README.md) -- if you have something to share
+- [Demos & open source](docs/README.md) -- if you have something to share
   about your use of CloudEvents, please submit a PR!
+- [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
 
-## Process
+### Security Concerns
 
-The CloudEvents project is working to formalize the
-[specification](cloudevents/spec.md) based on
-[design goals](cloudevents/primer.md#design-goals) which focus on
-interoperability between systems which generate and respond to events.
+If there is a security concern with one of the specifications in this
+repository please [open an issue](https://github.com/cloudevents/spec/issues).
 
-In order to achieve these goals, the project must describe:
```

Comment:
I removed this section because it felt old and kind of pointless since we describe the project at the top of the doc already.